images/tkg/docs/examples/adding_os_pkg_repos.md

```diff
@@ -0,0 +1,107 @@
+# Adding new OS packages and configuring the repositories or sources
+
+## Use case
+
+As a user I want to add new OS packages example `pkg1` and `pkg2` and also I want to configure ubuntu/photon sources/repositories to an internal mirror to build the node images in an air-gapped scenario. Configuring new sources/repositories is also useful when we want to install internally built software.
+
+## Customization
+
+OS packages and sources/repositories are configured through packer variables. To see the list of configurable packer variables provided by [kubernetes image builder][kubernetes-image-builder] please refer to this [page][customizations-doc].
```

Comment:
can we say

Configuration of OS packages and sources/repositories is exposed as packer variables. To view a list of packer variables exposed by kubernetes image-builder please refer